zeebe/clients/java/pom.xml

```diff
@@ -270,11 +288,12 @@
               <generatorName>java</generatorName>
               <inputSpec>${openapi.dir}/rest-api.yaml</inputSpec>
               <modelPackage>io.camunda.zeebe.client.protocol.rest</modelPackage>
+              <typeMappings>OffsetDateTime=String</typeMappings>
```

Comment:
:thought_balloon: I would strongly recommend using an appropriate type here and not just strings to avoid mismatches in general :shrug:

Reply:
Can't test the impact locally right now, unfortunately. If you think we should remove this from the client and it still works, I'm fine with that.

Reply:
I just added it for consistency, as otherwise the client may not send it as a string. I think it's OK, it's more for the Java side - we're enforcing RFC3339, but not using the type system to do that :shrug:

Reply:
OK, I see. I'll give this another look locally now to double-check. I get your point 👍

Reply:
OK, so, this is actually necessary now, thanks for adding it, @npepinpe. Otherwise, we cannot reset the date values. We need to send empty strings to the server to reset the values. If the client uses proper `OffsetDateTime`s, you cannot send empty strings. This wasn't necessary before because we used the free form attributes instead of the typed attributes of the request object.